conf/scalardb-custom-values.yaml

```diff
@@ -29,11 +35,26 @@ envoy:
                 operator: In
                 values:
                   - scalardbpool
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                    - scalardb
```

Comment:
Maybe better to use `scalardb-server`

Reply:
I have updated the name to scalardb-server.
Thank you